### DISCUSSION TITLE
Landing page issue - no user name PW fields on the landing page

### DISCUSSION THREAD

**Discussion by alexbash007** [kimai/kimai#3797] [On-Premise: Installation & Updates] [answered]
### Describe the issue

Hello, 

Freshly installed Kimai on a subdomain like so: kimai.<mydomain>.com. Went through the installation steps and all show success no errors, SQL base connection is fine too, I see the installer added tables, etc. However when I access the landing page kimai.<mydomain>.com I only see GOT NEW HOST and in the square below my subdomain kimai.<mydomain>.com. Am I missing something? Could you help? Thanks!

### I already tried

- [X] I've read and searched [the documentation](https://www.kimai.org/documentation/).
- [X] I've searched for similar issues in this repository.
- [X] I've searched for similar issues in [the discussions](https://github.com/kimai/kimai/discussions).

### Kimai version

1.30.5

### How do you run Kimai?

Other (please mention below)

### Which PHP version are you using?

7.4

**Comment by kevinpapst** [comment, marked as answer]
You have to point your domain to the `public/`directory as written in the documentation:
https://www.kimai.org/documentation/installation.html#webserver

**Reply by alexbash007** [reply]
Perfect, thank you I overlooked this! all I had to do is make the main landing page index.php file redirect to the /public/en/login subdirectory
